scripts/hid_configurator/modules/dfu.py

```diff
@@ -391,7 +414,9 @@ def dfu_start(dev, img_length, img_csum, offset):
     # verified by dfu sync at data exchange.
     event_data = struct.pack('<III', img_length, img_csum, offset)
 
-    success = dev.config_set('dfu', 'start', event_data)
+    dfu_module_name = dev.get_complete_module_name('dfu')
+    if dfu_module_name:
+        success = dev.config_set(dfu_module_name , 'start', event_data)
 
     if success:
```

Comment:
Variables refernced before assignment. Also in other places.
Causes runtime exceptions.